client/post-editor/editor-sidebar/index.jsx

```diff
@@ -77,6 +77,7 @@ const EditorSidebar = ( {
 							selectedRevisionId={ selectedRevisionId }
 							selectRevision={ selectRevision }
 							siteId={ site.ID }
+							type={ type }
```

Comment:
interesting, it's updated (I also updated #14927 to load `siteId` and `postId` from state).